test/kino_db/connection_cell_test.exs

```diff
@@ -60,6 +60,34 @@ defmodule KinoDB.ConnectionCellTest do
                {:ok, db} = Kino.start_child({Exqlite, opts})\
                """
     end
+
+    test "restores source code from attrs with BigQuery" do
+      attrs = %{
+        "variable" => "db",
+        "type" => "bigquery",
+        "project_id" => "",
+        "credentials" => %{},
+        "default_dataset_id" => ""
+      }
+
+      {_kino, source} = start_smart_cell!(ConnectionCell, attrs)
+
+      assert source ==
+               """
+               scopes = ["https://www.googleapis.com/auth/cloud-platform"]
+               credentials = %{}
+
+               goth_opts = [
+                 name: Goth,
```

Comment:
let's use something more unique, KinoDB.Goth or ReqBigQuery.Goth.

Reply:
I like the `ReqBigQuery.Goth` idea